src/helpers/themes/decisionColors.ts

```diff
@@ -25,14 +43,30 @@ const classificationColors = `
     ).join("")}
 `;
 
+const tagPromptColors = `
+  --correction-true: ${hatchedBackground("#dd0")};
+  --correction-false: ${hatchedBackground("#dd0")};
+`;
+
```

Comment:
I don't think the tag correction should be hatched - it's not really that special. It's like a verification or classification.